realscale.inx

```diff
@@ -16,16 +17,52 @@
     <item value="m">m</item>
     <item value="km">km</item>
   </param>
+  <param name="desc_unit" type="description">Large objects need a scale to be drawn on the page:</param>
+  <param name="showscale" type="boolean" _gui-text="Show Scale Ruler?">true</param>
+  <param name="drawnscale" type="enum" _gui-text="Drawing Scale:                    " _gui-description="-world unit for drawing">
+    <item value="1">1:1</item>
+    <item value="2">1:2</item>
+    <item value="4">1:4</item>
+    <item value="5">1:5</item>
+    <item value="8">1:8</item>
+    <item value="10">1:10</item>
+    <item value="16">1:16</item>
+    <item value="24">1:24</item>
+    <item value="25">1:25</item>
+    <item value="32">1:32</item>
+    <item value="33.3">1:33.3</item>
+    <item value="48">1:48</item>
+    <item value="50">1:50</item>
+    <item value="64">1:64</item>
+    <item value="96">1:96</item>
+    <item value="100">1:100</item>
+    <item value="128">1:128</item>
+    <item value="200">1:200</item>
+    <item value="250">1:250</item>
+    <item value="500">1:500</item>
+    <item value="1000">1:1000</item>
+    <item value="1250">1:1250</item>
+    <item value="2500">1:2500</item>
+  </param>
+  <param name="unitlength" type="enum" _gui-text="Unit length                                ">
+    <item value="1">1</item>
+    <item value="10">10</item>
+    <item value="100">100</item>
+  </param>
```

Comment:
Again, I'd prefer to enter a value here, so people aren't limited to what we think is best for them.